hal/src/stm32f2xx/core_hal_stm32f2xx.c

```diff
@@ -314,6 +319,11 @@ void HAL_Core_Config(void)
 #if PLATFORM_ID==8 // Additional pins for P1
     for (pin_t pin=24; pin<=29; pin++)
         HAL_Pin_Mode(pin, INPUT);
+    const uint8_t* data = (const uint8_t*)dct_read_app_data(DCT_RADIO_FLAGS_OFFSET);
+    uint8_t current = (*data);
+    if ((current&3) == 0x2) {
```

Comment:
See commit https://github.com/spark/firmware/pull/1120/commits/b3f8160de6428b597b2d35f576303dd46677abfc for resurrection of a fairy ;-)